### PR TITLE
Upgrade expensify-common

### PR DESCRIPTION
**Pull request by jasperhuangg**
### Details

Made some changes to ExpensiMark that were recently merged and deployed to npm, so this PR upgrades expensify-common so we can have those new changes.

If you're confused as to why the package.json isn't using the commit hash anymore, it's because I followed these steps (which I think use the major/minor version): https://stackoverflow.com/c/expensify/questions/7216

### Fixed Issues
N/A, but here's the expensify-common PR: https://github.com/Expensify/expensify-common/pull/376

### Tests
Same as QA

### QA Steps
Try to send this as a message in any chat, and verify it renders the same as the screenshot:

> \```hi\```
\```hi
\```
\```
hi\```
\```
hi
\```

Screen Shot 2021-05-24 at 8 38 41 PM

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

### Web
https://user-images.githubusercontent.com/31285285/119344670-84102b80-bcca-11eb-9903-8ac3907aad98.mp4

### Mobile Web
https://user-images.githubusercontent.com/31285285/119344719-90948400-bcca-11eb-80b9-5bc40a1ac858.mp4

### iOS
https://user-images.githubusercontent.com/31285285/119344749-99855580-bcca-11eb-84ac-090692a7aaf6.mp4

### Android
https://user-images.githubusercontent.com/31285285/119435608-0ccfab80-bd4d-11eb-9e4e-62fbabdb5358.mp4

### Desktop
https://user-images.githubusercontent.com/31285285/119344810-af931600-bcca-11eb-908f-69747c4200ff.mp4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,8 +79,8 @@
     "react-native-image-pan-zoom": "^2.1.12",
     "react-native-image-picker": "^2.3.3",
     "react-native-keyboard-spacer": "^0.4.1",
-    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#d0a5d12c08f2f029e3038fb3516a93a9403e9746",
     "react-native-modal": "^11.10.0",
+    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#d0a5d12c08f2f029e3038fb3516a93a9403e9746",
```
Comment (jasperhuangg) [inline package.json:83]: The version here wasn't changed (the commit hashes are the same), `npm install` simply changed the location of this when I ran it (I guess it sorted everything alphabetically).

**Comment by jasperhuangg** [comment]
@Luke9389 I self-reviewed just to clarify some stuff, this should be good for a quick review from you, thanks!

**Review by Luke9389 — APPROVED** [review]
Nice, simple version bump 👍

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "electron-log": "^4.2.4",
     "electron-serve": "^1.0.0",
     "electron-updater": "^4.3.4",
-    "expensify-common": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
+    "expensify-common": "^1.0.1",
```
Comment (marcaaron) [inline package.json:62]: We need to revert this. Sorry @jasperhuangg the SO instructions you followed are outdated.

Comment (Luke9389) [reply package.json:62]: Hey @marcaaron, can you add context on why this is being reverted? Just curious to see what broke.

Comment (marcaaron) [reply package.json:62]: We shouldn't be publishing `expensify-common` to npm. The advice in the SO is outdated (my bad). We must reference the commit hash instead like we have been doing for every other `expensify-common` update.

**Pull request merged**